apps/mobile/src/app/(home)/secure-your-wallet.tsx

```diff
@@ -24,7 +23,8 @@ export default function SecureYourWalletScreen() {
         style={{ paddingBottom: bottom + theme.spacing[5] }}
       >
         <AnimatedHeaderScreenLayout
-          rightTitleElement={<MoreInfoIcon onPress={() => {}} />}
+          // Hidden until linked
```

Comment:
Maybe you can add the ticker reference here?

Reply:
Good call! I have a reminder hanging in the Slack thread for this for days now. Will make one.